### PR TITLE
Add `inspec` and `kitchen-inspec` to ChefDK

### PR DESCRIPTION
**Pull request by schisamo**
/cc @fnichol @tyler-ball @danielsdeleo @chris-rock @arlimus @jamesc @jkeiser

### CODE REVIEW

**Comment by schisamo** [comment]
Sending a test build through now.

**Comment by tyler-ball** [comment]
:+1: if the build succeeds

**Comment by danielsdeleo** [comment]
As long as jenkins likes it :+1:

File: config/projects/chefdk.rb [hunk 1]
```diff
@@ -65,7 +65,12 @@
   override :rubygems,     version: "2.4.8"
 end
 
+# update to 0.9.1 when that is released
+# master currently has some dependency changes we need
+override :'inspec', version: "master"
+
 override :'test-kitchen', version: "v1.4.2"
+override :'kitchen-inspec', version: "master" # will set when release is tagged
```
Comment (arlimus) [inline config/projects/chefdk.rb]: Just pushed kitchen-inspec to rubygems: https://rubygems.org/gems/kitchen-inspec

Comment (schisamo) [inline config/projects/chefdk.rb]: Thanks @arlimus. We'll probably still pull things in via git (and a tag) for flexibility. Please create a `v0.9.0` tag in https://github.com/chef/kitchen-inspec. `rake release` will do this for you in the future:

```
rake release             # Create tag v0.9.0 and build and push kitchen-inspec-0.9.0.gem to Rubygems
```

**Pull request merged**